python/cuml/test/test_kmeans.py

```diff
@@ -239,25 +244,28 @@ def test_kmeans_sklearn_comparison_default(name, nrows):
 
     cu_y_pred = cuml_kmeans.fit_predict(X)
     cu_score = adjusted_rand_score(cu_y_pred, y)
-    kmeans = cluster.KMeans(random_state=12,
+    kmeans = cluster.KMeans(random_state=random_state,
                             n_clusters=params['n_clusters'])
     sk_y_pred = kmeans.fit_predict(X)
     sk_score = adjusted_rand_score(sk_y_pred, y)
 
     assert sk_score - 1e-2 <= cu_score <= sk_score + 1e-2
 
 
+@pytest.mark.parametrize(
+    'max_iter, oversampling_factor, max_samples_per_batch, init', [
+        (100, 0.5, 1 << 10, 'preset'),
+        (1000, 1.0, 1 << 15, 'preset'),
+        (500, 1.5, 1 << 5, 'k-means||'),
```

Comment:
Since `k-means||` is pretty important and has been a source of a couple of bugs in the past, I would add a second test similar to preset here